src/Core/Framework/App/Template/TemplateLoader.php

```diff
@@ -17,7 +17,7 @@ class TemplateLoader extends AbstractTemplateLoader
         'documents',
     ];
 
-    private const ALLOWED_FILE_EXTENSIONS = '*.html.twig';
+    private const ALLOWED_FILE_EXTENSIONS = '*.*.twig';
```

Comment:
it is, but i dont know why you develop a check for .html.twig files so i thought if there is any file which just ends with .twig and not <name>.**<extension>**.twig which should not be overwritten it would be necessary to do it like this. 

I cant really tell why you do specific check for .html.twig files because i have no template-file (.twig) in mind which should not be extendable by an app.

Reply:
IMO that should either be `'*.twig'` to allow all template files or explicitly `['*.html.twig', '*.xml.twig']` to only allow HTML and XML templates.

Reply:
We discussed this internally and suggest changing it to `*.twig`